# reason: https://github.com/raspberrypi/linux/issues/3024
# src: upstream

From e8a66b4f610b3a20bae8f706256d230135916c26 Mon Sep 17 00:00:00 2001
From: Dave Stevenson <dave.stevenson@raspberrypi.org>
Date: Fri, 28 Jun 2019 16:05:25 +0100
Subject: [PATCH] configs: Drop V4L2 camera and codec drivers from
 bcmrpi3_defconfig

They rely on mmal_vchiq, which in turn wants vc-sm-cma.
vc-sm-cma needs some attention for 64 bit, so drop it for now.

Signed-off-by: Dave Stevenson <dave.stevenson@raspberrypi.org>
---
 arch/arm64/configs/bcmrpi3_defconfig | 2 --
 1 file changed, 2 deletions(-)

diff --git a/arch/arm64/configs/bcmrpi3_defconfig b/arch/arm64/configs/bcmrpi3_defconfig
index 904163e4266dd..e81b099bd5420 100644
--- arch/arm64/configs/bcmrpi3_defconfig
+++ arch/arm64/configs/bcmrpi3_defconfig
@@ -1086,8 +1086,6 @@ CONFIG_FB_TFT_WATTEROTT=m
 CONFIG_FB_FLEX=m
 CONFIG_FB_TFT_FBTFT_DEVICE=m
 CONFIG_SND_BCM2835=m
-CONFIG_VIDEO_BCM2835=m
-CONFIG_VIDEO_CODEC_BCM2835=m
 CONFIG_MAILBOX=y
 CONFIG_BCM2835_MBOX=y
 # CONFIG_IOMMU_SUPPORT is not set
